javascript/javascript1/week4/js1-week4-homework/index.js

```diff
@@ -0,0 +1,92 @@
+const name = [];
+const toDo = [];
+function getReply(command) {
+  const lowerCase = command.toLowerCase();
+  //console.log(lowerCase)
+
+  let words = lowerCase.split(" ");
+  let myName = words[words.length - 1];
+  name.push(myName);
+  //console.log(name)
+  if (lowerCase.includes("hello my name is")) {
+    console.log("Nice to meet you " + name[0]);
```

Comment:
Technically the assignment says the function should return the string, not log it.